From: Boyuan Yang <073plan@gmail.com>
Date: Wed, 31 Aug 2016 14:45:32 +0800
Subject: README.txt: patch for documentation description.

Upstream README.txt fails to describe the correct status of
documentation delivered by Debian package. This patch gives
the correct description of documentation.
---
 README.txt | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/README.txt b/README.txt
index 18c46a2..d5ab3df 100644
--- a/README.txt
+++ b/README.txt
@@ -5,15 +5,16 @@ NixNote:: Evernote client clone for Linux
 
 This is an incomplete clone of Evernote designed to run on Linux. While this is designed to work with Evernote, it is in no way connected with or supported by Evernote. Any problems you encounter will not be corrected by them and, since this is GPL software, you are using this software at your own risk.
 
-See release.txt for details of what works and what doesn't work.
+See User Documentation for details of what works and what doesn't work.
 
-Documents:
+Documents in /usr/share/doc/nixnote2 directory:
 
 changelog.txt:  ChangeLog and development history
-install.txt:    Build and install instructions
-gpl.txt:        License description of GPL v2.
-license.txt:    Legal notices for licenses and trademarks
+shortcut_howto.txt: How-to document to setup shortcut keys.
+examples/colors.txt: A sample coniguration file to customize note bockground color options.
+help/UserDocumentation.odt: User Documentation in odt format.
+help/UserDocumentation.pdf: User Documentation in pdf format.
+
+Documents in /usr/share/nixnote2 directory:
 
 theme.ini: A file containing theme configuration options.
-shortcut_howto.txt: How-to document to setup shortcut keys.
-colors.txt: A sample coniguration file to customize note bockground color options.
